config.py

```diff
@@ -47,136 +55,255 @@
+        'default_lookup_name': '#ko_lastsync',
+        'config_label': _('Date Synced column:'),
+        'config_tool_tip': _('A "Date" column to store when the last sync was performed.'),
+        'data_source': 'sidecar',
+        'data_location': ['calculated', 'date_synced'],
+    },
+    'column_date_sidecar_modified': {
+        'column_heading': _("Date KOReader Modified"),
+        'datatype': 'datetime',
+        'description': _("Date when the book was last modified in KOReader. Wired sync only."),
+        'default_lookup_name': '#ko_lastmod',
+        'config_label': _('Date Modified column:'),
+        'config_tool_tip': _('A "Date" column to store when the sidecar file was last '
+                             'modified. Works for wired connection only, wireless will be '
+                             'always empty'),
+        'data_source': 'sidecar',
+        'data_location': ['calculated', 'date_sidecar_modified'],
+    },
+    'column_sidecar': {  # Unsure about Interpret this column as
+        'column_heading': _("KOReader Raw Sidecar"),
+        'datatype': 'comments',
+        'description': _("Raw sidecar data directly from KOReader. Allows sync to KOReader, also serves as a backup."),
+        'default_lookup_name': '#ko_sidecar',
+        'config_label': _('Raw sidecar column:'),
+        'config_tool_tip': _('A "Long text" column to store the contents of the\n'
+                             'metadata sidecar as JSON, with "Interpret this column as" set to\n'
+                             '"Plain text". This is required to sync metadata back to KOReader sidecars.'),
+        'data_source': 'sidecar',
+        'data_location': [],  # [] gives the entire sidecar dict
+        'transform': (lambda d: json.dumps(
+            {k: d[k] for k in d if k != 'calculated'},
+            skipkeys=True,
+            indent=2,
+            default=str
+        )),
+    },
+}
+
+CHECKBOXES = { # Each entry in the below dict is keyed with config_name
+    'checkbox_sync_if_more_recent': {
+        'config_label': 'Sync only if changes are more recent',
+        'config_tool_tip': 'Sync book only if the metadata is more recent. Requires\n'
+                '"Date Modified Column" or "Percent read column" to be synced',
+    },
+    'checkbox_no_sync_if_finished': {
+        'config_label': 'No sync if book has already been finished',
+        'config_tool_tip': 'Do not sync book if it has already been finished. Requires\n'
+                '"Percent read column" or "Reading status column" to be synced',
+    },
+    'checkbox_enable_automatic_sync': {
+        'config_label': 'Automatic Sync on device connection',
+        'config_tool_tip': 'Sync from KOReader automatically on device connection. \n'
+                'Restart calibre to apply this setting',
+    },
+    'checkbox_enable_scheduled_progressync': {
+        'config_label': 'Enable Daily ProgressSync',
```

Comment:
Lets make it `[ ] Daily ProgressSync` there no need to have extra `Enable` if checkbox represents boolean value